src/ports/postgres/modules/deep_learning/madlib_keras_helper.py_in

```diff
@@ -225,3 +227,19 @@ def query_weights(model_output_table, model_weights_col, mst_key_col, mst_key):
 
     res = plpy.execute(mlp_weights_query)
     return res[0][model_weights_col]
+
+def create_summary_view(model_table, mst_key):
+    tmp_view_summary = unique_string('tmp_view_summary')
+    model_summary_table = add_postfix(model_table, "_summary")
+    model_info_table = add_postfix(model_table, "_info")
+    if not (table_exists(model_summary_table) and
+            table_exists(model_info_table)):
+        plpy.error("{0}: Missing table".format(module_name))
+
+    plpy.execute("""
+        CREATE VIEW {tmp_view_summary} AS
+        SELECT *, 1::SMALLINT AS {model_arch_id_colname}
```

Comment:
I am assuming that we don't care about the model arch id and that's why it is always set to 1? If that's the case, I would suggest making it NULL so that the intention is clear and it will help prevent any regressions in the future

Reply:
The `validate_model_arch_table` requires that the model_id is not `None`.

Reply:
Maybe set it to -1 then and also add a brief comment.